lib/clif-backend/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "wasmer-clif-backend"
+version = "0.1.0"
+authors = [
+    "Lachlan Sneff <lachlan.sneff@gmail.com>",
+    "Steve Akinyemi <steve@wasmer.io>",
+    "Syrus Akbary <syrus@wasmer.io>"
+]
```

Comment:
should this be Wasmer developers? Seems like we are past a few contributors now.

Reply:
Oh good idea, we should change all the Cargo.toml files to be `The Wasmer Developers <wasmer.io>`

Reply:
I've just created the email group `engineering@wasmer.io`.

We can use it as the author of the package: `Wasmer Engineering <engineering@wasmer.io>`.